app/controllers/users_controller.rb

```diff
@@ -19,26 +19,14 @@ def new_with_invitation_token
 
   def create
     @user = User.new(user_params)
+    result = UserSignup.new(@user).sign_up(params[:stripeToken], params[:invitation_token])
```

Comment:
One good way how to pass arguments to methods would be to have hash of options, so you don't have to remember the order of arguments.

```ruby
UserSignup.new(@user).sign_up(
  stripe_token: params[:stripeToken],
  invitation_token: params[:invitation_token])
```
Another advantage is, you don't have to pass `nil` as well. If you don't want to pass the invitation token for instance, you can just leave it out.